frontend/src/main/web/src/components/Base/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { PropTypes } from 'react'
+import warning from 'warning'
+import { merge } from 'lodash'
+import { flattenThemeClasses } from '../../utils/styleUtils'
+
+const Base = ({
```

Comment:
Check the readme.md file on each of the component directory.


---------------------------------------------

Alex Eng
Senior Software Engineer
Globalisation Tools Engineering
DID: +61 3514 8262 <callto:+61+3514+8262>
Mobile: +614 2335 3457 <callto:+614+2335+3457>

Red Hat, Asia-Pacific Pty Ltd
Level 1, 193 North Quay
Brisbane 4000
Office: +61 7 3514 8100 <callto:+61+7+3514+8100>
Fax: +61 7 3514 8199 <callto:+61+7+3514+8199>
Website: www.redhat.com

On Mon, Apr 18, 2016 at 2:27 PM, David Mason <notifications@github.com>
wrote:

> In frontend/src/main/web/src/components/Base/index.jsx
> <https://github.com/zanata/zanata-server/pull/1125#discussion_r60004933>:
>
> > @@ -0,0 +1,80 @@
> > +import React, { PropTypes } from 'react'
> > +import warning from 'warning'
> > +import { merge } from 'lodash'
> > +import { flattenThemeClasses } from '../../utils/styleUtils'
> > +
> > +const Base = ({
>
> What is a Base? This needs a descriptive comment.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1125/files/9ea162561d4c46d4b7aeb5cdbba300cd52235452#r60004933>
>